pyproject.toml

```diff
@@ -20,13 +20,10 @@ python = "^3.6"
 actfw-gstreamer = "*"
```

Comment:
Note that I leave this because `actfw-gstreamer` seems to be under rc yet.
This is TODO.